app/frontend/src/setupTests.ts

```diff
@@ -13,6 +13,7 @@ global.localStorage = createLocalStorageMock();
 
 afterEach(() => {
   global.localStorage.clear();
+  jest.restoreAllMocks();
```

Comment:
Pretty much every test suite has an/some error case(s) that lead(s) to `console.error` needing stubbed out and being reset so that other tests still can log errors and allow us to see problems, so I thought worth moving it here so it's done automatically for all test suites by default.